packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -4,8 +4,15 @@
 import { Button, Modal } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 
-export const convertDescription = __(
-	'This page list is synced with the published pages on your site. Detach the page list to add, delete, or reorder pages yourself.'
+import { createInterpolateElement } from '@wordpress/element';
+
+export const convertDescription = createInterpolateElement(
+	__(
+		"This menu is currently displaying your website's pages. Editing it will allow you to add, delete, or reorder pages, but new pages will no longer be <em>automatically</em> added to this navigation."
```

Comment:
```suggestion
		"This navigation menu is currently displaying your website's pages. Editing it will allow you to add, delete, or reorder pages, but new pages will no longer be <em>automatically</em> added."
```

Ok I removed the second reference to the "menu". Does it work?